morseMessenger/flashHook.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { morseDictionary, DictOptions } from './morseDictionary';
+
+export default function useFlash() {
+  const TORCH_ON = 'torch';
+  const TORCH_OFF = 'off';
+  const PAUSE = 1000;
+  const FLASH_PAUSE = 400;
+  const FLASH_SHORT = 100;
+  const FLASH_LONG = 700;
```

Comment:
I'd put these consts above `useFlash`

It's a bit cleaner to separate consts from things which are local as then you mark: "hey, these things are constant variables used for my algorithm's logic. You don't need to look at them."